filters/ThirdParty/filter_210_Spam404/template.txt

```diff
@@ -5,3 +5,5 @@
 @@||sentry.mba^$~third-party
 ! https://forum.adguard.com/index.php?threads/nulled-to-incorrect-blocking.30024
 @@||nulled.to^$domain=nulled.to
+! https://forum.adguard.com/index.php?threads/dropbbase-com-incorrect-blocking.31248/
+@@||dropbbase.com^$domain=dropbbase.com
```

Comment:
I am not sure, if we should to fix this issue. It is expected result for this filter.

Reply:
Users can download items from it (and a text file is not spam all the time imho), so I do not see it as a Spam.

Reply:
ok